soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/algorithm/ConcurrentRateLimiterAlgorithm.java

```diff
@@ -38,4 +43,11 @@ protected String getScriptName() {
     protected String getKeyName() {
         return "concurrent_request_rate_limiter";
     }
+
+    @Override
+    public List<String> getKeys(final String id) {
+        String prefix = getKeyName() + ".{";
+        String tokenKey = prefix + "}.tokens";
```

Comment:
the key can used uuId?  NOT request id

Reply:
Does RateLimiterPlugin generate a UUID and pass it in? Because RateLimiterPlugin needs to remove by this ID